### DISCUSSION TITLE
[Feature] Mobile Upload albums into existing Immich Albums

### DISCUSSION THREAD

**Discussion by MoldyTaint** [immich-app/immich#2077] [Feature Request]
### The feature

The ability to select an Album on your phone's Immich app and sync it into an already existing Album what Immich server hosts.

For example:

Album on phone is labled "Family Photos"

Sync it to the Album stored on the server labled "Random Phone Pics"

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1678
